perllib/FixMyStreet/Cobrand/UKCouncils.pm

```diff
@@ -31,11 +31,6 @@ sub restriction {
     return { cobrand => shift->moniker };
 }
 
-sub problems_restriction {
-    my ($self, $rs) = @_;
-    return $rs->to_body($self->council_id);
-}
```

Comment:
Thinking about it some more, perhaps changing Nearby/Around/etc would be better after all. I guess we have the following areas, for UK council cobrands:
* Admin, viewing all problems restricted to a cobrand;
* Around pages, viewing all problems;
* "Report" pages (either individual or list format), viewing cobrand-only problems - that's Report, Photo, Open311, My, JSON (think that's it).
In another scenario, you could imagine cobrands all being able to view all problems, the cobrand is just for rebranding purposes, or for a cobrand to only want to view its problems. So I guess adding another problems lookup for around pages might be easiest? Not sure of best names. problems, admin_problems, and map_problems? Though 'map_problems' would be problems suitable for showing on a map, not the ones to actually show on the current map, hmm.

There are other bugs in this area I've just discovered (e.g. /rss/problems on a cobrand doesn't respect any problems restriction currently).